index.html

```diff
@@ -28,180 +34,240 @@ <h1 class="header">I'am Sentayhu <br>  Glad to see <br>you!</h1>
+    <h2 class="tonic1">Tonic</h2>
+<div class="mobile-hidden tonic2">Facebook 360</div>
+ <br>
+    <div class="social-mv">
+     <ul class="connect-via card-info desktop-mv mobile-mv">
          <li class="canopy">Canopy </li>
      <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp;&nbsp; Back End Dev</li>
-     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters">&nbsp; &nbsp;2015</li>
-     </ul>
+     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp; &nbsp; 2015</li>
+     </ul></div>
      <div class="discription">
        <p class="discpara">A daily selection of 
            privately personalized reads; no
             accounts or sign-ups required.</p>
      </div>
-     <ul class="connect-via">
+     <div class="social-mv">
+     <ul class="connect-via desktop-mv-lang">
          <li class="lang">HTML</li>
          <li class="lang">CSS</li>
          <li class="lang">JavaScript</li>
-     </ul>
+     </ul> </div>
      <button type="button" class="project1-btn">See Project</button>
     </div>
 </div>
+</div>
 <!-- ** END OF CARD 3 **-->
 <div class="subwork">
-    <div class="worksec1">
+    <div class="worksec1 worksec1d">
+        <div class="worksf" >
     <img src="./assets/images/Snapshoot%20Portfolio3.svg" class="cardpic" alt="pic">
-    <div class="card"></div>
-    <h1 class="tonic1">Multi-Post Stories</h1> 
-     <ul class="connect-via card-info">
+    
+</div>
+<!-- work sec start******** -->
+<div class="worksf">
+    <div class="mobile-hidden tonic2">Uber Navigation</div>
+    <div class="desktop-hidden">
+        <h2 class="tonic1">Multi-Post Stories</h2></div>
+    <div class="social-mv">
+     <ul class="connect-via card-info desktop-mv mobile-mv-1">
          <li class="canopy">Canopy </li>
      <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp;&nbsp; Back End Dev</li>
-     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters">&nbsp; &nbsp;2015</li>
-     </ul>
+     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp; &nbsp; 2015</li>
+     </ul></div>
      <div class="discription">
        <p class="discpara">A daily selection of 
            privately personalized reads; no
             accounts or sign-ups required.</p>
      </div>
-     <ul class="connect-via">
+     <div class="social-mv">
+     <ul class="connect-via desktop-mv-lang">
          <li class="lang">HTML</li>
          <li class="lang">CSS</li>
          <li class="lang">JavaScript</li>
-     </ul>
+     </ul> </div>
      <button type="button" class="project1-btn">See Project</button>
     </div>
+    <div class="worksf">
+        <img src="./assets/images/Snapshoot%20Portfolio1.svg" class="cardpic1" alt="pic">
+    </div>
+</div>
 </div>
 </section>
 <br><br>
 <section  class="about">
```

Comment:
- [ ] As pointed out by the previous reviewer, please kindly make sure your design  in the about section matches the original figma design guidelines(fonts, colors, margins, padding etc)

### Your Design
![image](https://user-images.githubusercontent.com/43995830/154553912-51f93cbc-48dc-4b21-86e5-2101b7c4ef48.png)

###  Original Design
![image](https://user-images.githubusercontent.com/43995830/154554128-142d8002-ca7d-4e18-bb62-6d1fb0f3ea4c.png)